CONTRIBUTING.md

```diff
@@ -20,3 +20,19 @@ If you added a contributor by mistake, you can remove them in a comment with:
 
 If you are making a pull request on behalf of someone else but you had no part in designing the 
 feature, you can remove yourself with the above syntax.
+
+# Tests
+
+For normative changes for any specification in
+[CR or later](https://www.w3.org/Style/CSS/current-work) as well as the pre-CR specifications listed
+below, a corresponding [web-platform-tests](https://github.com/w3c/web-platform-tests) PR must be
+provided; for other specifications it is highly appreciated. Typically, both PRs will be merged at
```

Comment:
“for other specifications it is highly appreciated” I would probably downgrade this a bit to maybe s/highly/often/. I would not have appreciated people submitting tests to CSS Grid in the earlier exploratory stages of the spec: that just means more work for for us to go through the for-sure outdated and possibly entirely useless tests later on--and discourages submissions without tests even in cases where they wouldn't actually be helpful.